src/app.rs

```diff
@@ -393,6 +407,18 @@ impl App {
                 || self.matches.value_of("color") == Some("always")
                 || self.matches.value_of("decorations") == Some("always")),
             files,
+            tab_width: self
+                .matches
+                .value_of("tabs")
+                .and_then(|w| w.parse().ok())
+                .or_else(|| env::var("BAT_TABS").ok().and_then(|w| w.parse().ok()))
```

Comment:
could the `.and_then(|w| w.parse().ok())` part be used only once? Or are we dealing with different string types here?

Reply:
It's implemented this way so that an invalid value for --tabs would fall back on the environment variable. Should it be done differently?

Reply:
I think we should fail with an error message if the value for `--tabs` can not be parsed (instead of silently falling back on the environment-variable value).

Reply:
Done in d404139